libs/core/src/lib/table/table.component.scss

```diff
@@ -23,3 +23,7 @@
 		white-space: nowrap;
 	}
 }
+
+.fd-table__row--unnavigatable {
```

Comment:
Add this to fundamental styles, and when it's released, remove it from here

Reply:
PR is up https://github.com/SAP/fundamental-styles/pull/2411. I'll remove the changes in this file once the new stylesheet is released.

Reply:
This change is no longer required. The latest commit has taken it out.